electrumsv/wallet_database/tables.py

```diff
@@ -794,6 +794,34 @@ class TransactionDeltaTable(BaseWalletStore):
         "FROM TransactionDeltas AS TD "
         "INNER JOIN KeyInstances AS KI ON TD.keyinstance_id = KI.keyinstance_id AND "
             "KI.account_id = ?")
+    READ_CANDIDATE_USED_KEYS = ("""
+        WITH constants AS (
+            SELECT 1<<0 AS is_active_flag,
```

Comment:
Can't you use an multi-line f-string and put these constants directly from KeyInstanceFlags? e.g. f"""blah blah {KeyInstanceFlag.IS_ACTIVE} blah blah"""